.github/workflows/ci.yml

```diff
@@ -6,10 +6,11 @@ on:
   pull_request:
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Looks like prettier is failing on the windows build. I found this comment, which suggests line endings may be the issue: https://github.com/prettier/prettier/issues/7825

Perhaps we should add a `.gitattributes` file like it suggests?